commands/user.go

```diff
@@ -402,3 +412,20 @@ func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error
 
 	return nil
 }
+
+func verifyUserCmdF(c client.Client, cmd *cobra.Command, userArgs []string) error {
+	users := getUsersFromUserArgs(c, userArgs)
+	for i, user := range users {
+		if user == nil {
+			printer.PrintError(fmt.Sprintf("can't find user '%v'", userArgs[i]))
+			continue
+		}
+
+		_, resp := c.VerifyUser(user.Id)
```

Comment:
I'm still thinking that we should find a better name given that we already have a `VerifyUserEmail` method in the server client :sweat_smile: 

Also, what is the purpose of returning the `model.User` in the API and client methods? Are we going to use it or something?

Reply:
I agree with you that the new method could be used `VerifyUserEmail`. But its taken 😅  If we modify it, it would be a breaking change in the `client4.go` therefore we can't modify `VerifyUserEmail`. 
For the return, since we are updating a entity I thought we might want to return the entity, but I'm 1/5 on that.

Reply:
Agree, actually we should print the updated user for the `--format json` calls:

```suggestion
		if newUser, resp := c.VerifyUser(user.Id); resp.Error != nil {
			printer.PrintError(fmt.Sprintf("unable to verify user %s email: %s", user.Id, resp.Error))
		} else {
			printer.PrintT("User {{.Username}} verified", newUser)
		}
```